apps/core/app/[locale]/(default)/cart/_components/shipping-estimator/get-shipping-countries.ts

```diff
@@ -2,7 +2,8 @@ import { getCountries } from '~/client/management/get-countries';
 import { getShippingZones } from '~/client/management/get-shipping-zones';
 
 export const getShippingCountries = async () => {
-  const shippingZones = await getShippingZones();
+  const [shippingZones, allCountries] = await Promise.all([getShippingZones(), getCountries()]);
```

Comment:
Am I missing something, or should these not be invoked 🤔 
```suggestion
  const [shippingZones, allCountries] = await Promise.all([getShippingZones, getCountries]);
```